src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -205,8 +229,29 @@ function WorkspacesListRow({
                     </Text>
                 </View>
             </View>
+            {isJoinRequestPending && !isNarrow && (
+                <View style={[styles.flexRow, styles.gap2, styles.alignItemsCenter, styles.flex1, styles.justifyContentEnd, styles.mln6, styles.pr4]}>
+                    <View style={[styles.flexRow, styles.alignItemsCenter, styles.border, styles.pv1, styles.pl1, styles.pr2]}>
```

Comment:
The border radius should be `theme.radius-sm` (4px).

(I know we just received the design specs, so just noting just it down)

<img width="836" alt="image" src="https://github.com/Expensify/App/assets/29673073/89f9c5d9-e10a-4e50-9c05-d6c0f93081bc">